### PR TITLE
Add android services support

### PR DESCRIPTION
**Pull request by Heezay**

### CODE REVIEW

**Comment by Heezay** [comment]
We can parse services into manifest now.

File: crossbundle/cli/src/commands/build/build_context.rs [hunk 1]
```diff
@@ -39,9 +35,11 @@ impl BuildContext {
             .custom_metadata()
             .ok_or(Error::InvalidManifestMetadata)?
             .to_owned();
+        println!("custom_metadataa is {:?}", custom_metadata);
         let metadata = custom_metadata
             .try_into::<Metadata>()
             .map_err(|_| Error::InvalidManifestMetadata)?;
+        println!("metadata is {:?}", metadata);
```
Comment (enfipy) [inline crossbundle/cli/src/commands/build/build_context.rs:42]: Remove printlns

File: examples/bevy-2d/Cargo.toml [hunk 2]
```diff
@@ -11,7 +11,7 @@ anyhow = "1.0"
 bevy = { version = "0.7.0", features = ["mp3"] }
 
 [package.metadata]
-app_name = "Bevy 2D"
+app_name = "Bevy2D"
```
Comment (enfipy) [inline examples/bevy-2d/Cargo.toml:14]: Name should be with space

File: examples/bevy-2d/Cargo.toml [hunk 3]
```diff
@@ -24,3 +24,13 @@ android_res = "res/android"
 apple_build_targets = ["aarch64-apple-ios", "x86_64-apple-ios"]
 apple_assets = "assets"
 apple_res = "res/apple"
+# android_service = [{ name = "UpdateService", intent_filter = [], meta_data = []}]
```
Comment (enfipy) [inline examples/bevy-2d/Cargo.toml:27]: Why commented?

File: crossbundle/cli/src/commands/build/build_context.rs (hunk 1 above)
Comment (Heezay) [reply crossbundle/cli/src/commands/build/build_context.rs:42]: Not All features I wanted to implement were done. I still need it to debug.

File: examples/bevy-2d/Cargo.toml (hunk 3 above)
Comment (Heezay) [reply examples/bevy-2d/Cargo.toml:27]: Unused code. Will remove it.

**Review by enfipy — APPROVED** [review]

**Pull request merged**